client/styles/components/_forms.scss

```diff
@@ -1,3 +1,10 @@
+.form {
+  width:90%;
```

Comment:
rather than doing a percent width here, i think it makes sense to add a fixed padding or margin.